src/main/java/org/example/springbootintro/service/order/OrderServiceImpl.java

```diff
@@ -81,7 +81,7 @@ public OrderDto update(Long id, UpdateOrderItemDto requestDto) {
         return orderMapper.toDto(orderRepository.save(order));
     }
 
-    private Set<OrderItem> getOrderItems(ShoppingCart shoppingCart, Order order) {
+    public Set<OrderItem> getOrderItems(ShoppingCart shoppingCart, Order order) {
```

Comment:
if it was just the private method that was used only inside other methods of the same class - we should not make it public to test it, just write a test cases for the methods that used it to verify it's work